material-popup-menu/src/main/java/androidx/appcompat/widget/MaterialRecyclerViewPopupWindow.kt

```diff
@@ -359,4 +371,12 @@ class MaterialRecyclerViewPopupWindow(
 
         return menuWidth
     }
+
+    private fun addBackgroundDimming() {
```

Comment:
Cool, didn't know that adding a dim can be done with few lines of code like this!